src/ui/screen.rs

```diff
@@ -133,12 +133,20 @@ impl StatusArea {
         )?;
 
         let custom_info = if !custom_info.trim().is_empty() {
-            format!("━ {} ", custom_info.trim())
+            format!("━ {}{} ", custom_info.trim(), color::Fg(color::Green))
```

Comment:
Do a color::Fg(color::Reset) here before switching to Green. Just to make sure all the "flags" are cleaned first.